packages/react/src/stories/ic-chip.stories.mdx

```diff
@@ -463,7 +463,7 @@ export const defaultArgs = {
   disabled: false,
   dismissible: false,
   label: "Default",
-  size: "default",
+  size: "medium",
```

Comment:
I've just looked at Chip and Badge, assume this goes for all playgrounds, but the size control is still showing as small/default/large

<img width="778" alt="Screenshot 2024-08-27 at 10 41 10" src="https://github.com/user-attachments/assets/7f9fd6f7-4425-4f56-83da-6944b2a2af8a">